src/root/user.tt

```diff
@@ -80,6 +80,7 @@
           <select multiple="multiple" name="roles" class="span3" [% IF !c.check_user_roles('admin') %]disabled="disabled"[% END %]>
             [% INCLUDE roleoption role="admin" %]
             [% INCLUDE roleoption role="create-projects" %]
+            [% INCLUDE roleoption role="restart-jobs" %]
```

Comment:
I _believe_ this role will be created on save: https://github.com/NixOS/hydra/blob/master/src/lib/Hydra/Controller/User.pm#L217-L218 and also I see nowhere in the code which explicitly adds the existing roles to the DB.